commands/applications.go

```diff
@@ -21,9 +21,9 @@ func NewApplications(network *net.Network) (cmd *Applications) {
 
 func (cmd *Applications) Metadata() command_metadata.CommandMetadata {
 	return command_metadata.CommandMetadata{
-		Name:        "applications",
+		Name:        "application",
 		Description: "Show the status and location of running applications",
-		Usage:       "BROOKLYN_NAME applications",
```

Comment:
In this branch the 'Application's class has moved to being a subcommand of 'list', so the BROOKLYN_NAME isn't actually right there.  The help text for list grabs the usage for each of its subcommands, so the help for list now looks like:
```
NAME:
   list - List details for a variety of operands

USAGE:
   bk list (application | entity | sensor | effector)
... list application			Show the status and location of running applications
... list entity APPLICATION		Show the entites for an application
... list sensor APPLICATION ENTITY	Show the sensors for an application and entity
... list effector APPLICATION ENTITY	Show the list of effectors for an application and entity
```